tests/ui/pages/desktop/base.py

```diff
@@ -26,6 +26,15 @@ def login(self, email, password):
     def logout(self):
         self.header.click_logout()
 
+    def wait_for_page_to_load(self):
+        body = self.find_element(By.TAG_NAME, 'body')
+        self.wait.until(lambda s: 'firefox' in body.get_attribute('class'))
```

Comment:
It is specific. I don't see another attribute to wait for that would make some sense. If geckodriver would just wait..... 😡